### DISCUSSION TITLE
Questions about OS compatibility

### DISCUSSION THREAD

**Discussion by FengZh61** [vectordotdev/vector#16361] [Q&A] [answered]
Which version of vector can run on Linux 5.7 with kernel version 2.6.18-274.el5?
My server has several ancient but still productive!

**Comment by jszwedko** [comment, marked as answer]
Hi @FengZh61 !

Vector's only dynamic library dependency is libc. The required version there has varied but is currently version 2.17 for the x86_64 architecture  (as of Vector v0.27.0).

If you are using the `host_metrics` source there may be other assumptions about the host system. The library we use for that documents support for kernel 3.2+ but may work fine on older kernels.

**Reply by FengZh61** [reply]
> Hi @FengZh61 !
> 
> Vector's only dynamic library dependency is libc. The required version there has varied but is currently version 2.17 for the x86_64 architecture (as of Vector v0.27.0).
> 
> If you are using the `host_metrics` source there may be other assumptions about the host system. The library we use for that documents support for kernel 3.2+ but may work fine on older kernels.

[vector-0.12.2-x86_64-unknown-linux-musl.tar.gz] can run on my system.
I have no plans to upgrade it, waiting for his business to go offline and just recycle it, but for now I still need to capture its logs, vector is great and the compiled file offers many versions!
